lms/djangoapps/instructor/views/api.py

```diff
@@ -2708,7 +2704,7 @@ def start_certificate_regeneration(request, course_id):
         )
 
     # Check if the selected statuses are allowed
-    allowed_statuses = GeneratedCertificate.get_unique_statuses(course_key=course_key, flat=True)
+    allowed_statuses = [CertificateStatuses.downloadable, CertificateStatuses.error, CertificateStatuses.notpassing]
```

Comment:
@asadiqbal08  Certificate regenerate UI now has the following options

1. Regenrate for learners who have already received certificates (#)
2. Regenerate for leaners who have not received certificates (#)
3. Regenerate for learners in an error state (#)

that is why I have hard coded the allowed statuses as they will be same for all requests.